app/models/criterion.rb

```diff
@@ -100,5 +108,16 @@ def self.update_assigned_groups_counts(assignment, criterion_ids = nil)
         WHERE assignment_id = #{assignment.id}
           #{"AND id IN (#{criterion_ids_str})" unless criterion_ids_str.empty?}
     UPDATE_SQL
+    RubricCriterion.connection.execute(<<-UPDATE_SQL)
+      UPDATE #{RubricCriterion.table_name} AS c SET assigned_groups_count =
+        (SELECT count(DISTINCT g.id) FROM memberships AS m
+          INNER JOIN groupings AS g ON m.grouping_id = g.id
+          INNER JOIN criterion_ta_associations AS ct ON m.user_id = ct.ta_id
+          WHERE g.assignment_id = #{assignment.id}
+            AND ct.criterion_id = c.id AND ct.assignment_id = c.assignment_id
+            AND m.type = 'TaMembership')
+        WHERE assignment_id = #{assignment.id}
+    #{"AND id IN (#{criterion_ids_str})" unless criterion_ids_str.empty?}
+    UPDATE_SQL
```

Comment:
@WChrisK I made a few changes to self.update_assigned_groups_counts, so assuming the changes are approved you can replicate what I did for Flexible and Rubric.